internal/naming/selectors.go

```diff
@@ -88,17 +88,23 @@ func ClusterPatronis(cluster *v1beta1.PostgresCluster) metav1.LabelSelector {
 	}
 }
 
+// ClusterPostgresUsers selects things labeled for PostgreSQL users in cluster.
+func ClusterPostgresUsers(cluster string) metav1.LabelSelector {
+	return metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			LabelCluster: cluster,
+		},
+		MatchExpressions: []metav1.LabelSelectorRequirement{
+			// The now-deprecated default PostgreSQL user secret lacks a LabelRole.
+			// The existence of a LabelPostgresUser matches it and current secrets.
```

Comment:
> The existence of a LabelPostgresUser matches it and current secrets.

:question: I'm not sure what this comment is trying to say. Should this selector be able to grab a deprecated user secret? iiuc, LabelPostgresUser will only work for new secrets

Reply:
I renamed the constant from `LabelUserSecret` to `LabelPostgresUser`. Both old and new have it; the value is the Postgres user name.

Reply:
Ah, I see. Sorry didn't catch that